docs/searchpredicates.adoc

```diff
@@ -14,6 +14,10 @@ The `Compare` enum specifies the following comparison predicates used for index
 * `lt` (less than)
 * `lte` (less than or equal)
 
+All those predicates can be used on String, numbers, Date, Instant.
+
+`eq` and `neq` can be used on Boolean and UUID.
+
```

Comment:
Maybe "All comparison predicates are supported by String, numeric, Date and Instant data types. Boolean and UUID data types support the `eq` and `neq` comparison predicates."